src/motion/motion.ts

```diff
@@ -52,8 +56,10 @@ export class Motion implements vscode.Disposable {
             let selection = e.selections[0];
 
             if (selection) {
-                let line = selection.active.line;
-                let char = selection.active.character;
+                const whosFirst = selection.anchor.compareTo(selection.active);
```

Comment:
Um... actually, not totally sure what I was doing here o_O

Gonna revert this.